scripts/install/deploy/install_redis.sh

```diff
@@ -30,3 +26,6 @@ if [ "$?" -gt "0" ]; then
 else
   echo "Redis already running"
 fi
+
+sudo cp redis.service /lib/systemd/system/
+sudo systemctl enable redis
```

Comment:
@paulodamaso Shouldn't these Redis scripts also be added to the Docker image? As I recall, we have setup Docker at some point.

Reply:
@amihaiemil I thought that initially, but @emilianodellacasa said no: https://github.com/timelesslounge/timelessis/issues/300#issuecomment-465233283